db_R_tutorial.Rmd

```diff
@@ -63,11 +63,11 @@ if(!require(DBI)){
   library(DBI)
 }
 
-# # load RSQLite to make database engine in R and provides an interface compliant with the DBI package
-# if(!require(RSQLite)){
-#   install.packages("RSQLite")
-#   library(RSQLite)
-# }
+# load RSQLite to make database engine in R and provides an interface compliant with the DBI package
+if(!require(RSQLite)){
```

Comment:
Do you run into problems when you don't load RSQLite?

Reply:
sorry, now I think I understand. If you want to make sure that all packages are installed it should be in there.
I would put it before the segment with DBI though because RSQLite should install DBI automatically...

Reply:
Yes, that's the way.